src/OrchardCore.Modules/OrchardCore.Contents/Views/Content.SummaryAdmin.cshtml

```diff
@@ -57,3 +57,15 @@
 {
     <div class="col primary">@await DisplayAsync(Model.Content)</div>
 }
+
+<script at="Foot" type="text/javascript">
```

Comment:
Why not use a named script with a class selector ? Instead of having tons of instances of this code on the page ?

Reply:
@netwavebe already fixed it @jptissot exactly as you suggested.

Reply:
Ah cool, I did not realize this was an old pr :)

Reply:
It got merged to dev a few days ago. See https://github.com/OrchardCMS/OrchardCore/pull/7070

Reply:
Thanks @netwavebe :)